cmd/dep/status.go

```diff
@@ -150,6 +158,123 @@ func (out *jsonOutput) MissingFooter() {
 	json.NewEncoder(out.w).Encode(out.missing)
 }
 
+type dotProject struct {
```

Comment:
I think everything related to generating graphviz output could be safely put into its own, separate file - `graphviz.go`